pkg/apis/core/v1beta1/defaults_shoot.go

```diff
@@ -329,6 +329,9 @@ func SetDefaults_ClusterAutoscaler(obj *ClusterAutoscaler) {
 	if obj.MaxGracefulTerminationSeconds == nil {
 		obj.MaxGracefulTerminationSeconds = pointer.Int32(600)
 	}
+	if obj.IgnoreDaemonsetsUtilization == nil {
```

Comment:
Enhance test for this new case [here](https://github.com/gardener/gardener/blob/ccb71cb5fa98b95be02e6335415905078291bed0/pkg/apis/core/v1beta1/defaults_shoot_test.go#L863).